package.json

```diff
@@ -24,15 +24,15 @@
     "docker:build:head:cross": "npm run build:cross && gulp docker-image:head:cross",
     "docker:push:head:cross": "npm run docker:build:head:cross && gulp push-to-docker:head:cross",
     "docker:push:release:cross": "npm run docker:build:release:cross && gulp push-to-gcr:head:cross",
+    "cypress:open": "npx cypress open",
     "test": "npm run test:frontend && npm run test:backend && npm run test:e2e",
     "test:frontend": "ng test --karma-config ./aio/karma.conf.js --watch=false",
     "test:frontend:coverage": "ng test --karma-config ./aio/karma.conf.js --watch=false --code-coverage",
     "test:frontend:watch": "ng test --karma-config ./aio/karma.conf.js",
     "test:backend": "go test github.com/kubernetes/dashboard/src/app/backend/...",
     "test:backend:coverage": "./aio/scripts/coverage-backend.sh",
     "test:coverage": "npm run test:frontend:coverage && npm run test:backend:coverage",
-    "test:e2e": "ng e2e",
-    "pretest:e2e": "webdriver-manager update --standalone false --versions.chrome 2.45 --gecko false",
+    "test:e2e": "npx cypress run",
```

Comment:
I'd change it to `e2e` so we can use `npm e2e` from now. `e2e` are not the standard test.